### PR TITLE
View creator api public release

### PR DESCRIPTION
**Pull request by roopksaini**
APIs tagged as public (vs. beta)

### CODE REVIEW

File: core/frontend/src/ViewCreator2d.ts [hunk 1]
```diff
@@ -32,7 +32,8 @@ export interface ViewCreator2dOptions {
   useSeedView?: boolean;
```
Comment (pmconne) [inline core/frontend/src/ViewCreator2d.ts:32]: Elaborate on what this does. How is "seed" view selected?

File: core/frontend/src/ViewCreator2d.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ import { DrawingViewState } from "./DrawingViewState";
 import { SheetViewState } from "./SheetViewState";
 import { EntityState, loggerCategory } from "./imodeljs-frontend";
 
-/** @beta Options for creating a 2d [[ViewState]] via [[ViewCreator2d]] */
+/** @public Options for creating a 2d [[ViewState]] via [[ViewCreator2d]] */
```
Comment (pmconne) [inline core/frontend/src/ViewCreator2d.ts:25]: Put release tags on separate (last) line of doc comments.

File: core/frontend/src/ViewCreator2d.ts [hunk 3]
```diff
@@ -32,7 +32,8 @@ export interface ViewCreator2dOptions {
   useSeedView?: boolean;
 }
 
-/** @beta API for creating a [[ViewState]] for a 2D model ([[GeometricModel2dState]]).
+/**
+ * @public API for creating a [[ViewState]] for a 2D model ([[GeometricModel2dState]]).
```
Comment (pmconne) [inline core/frontend/src/ViewCreator2d.ts:36]: `[[ViewState]]` => `[[ViewState2d]]`.

Comment (pmconne) [reply core/frontend/src/ViewCreator2d.ts:36]: `[[iModelConnection]] is not a valid link.

**Review by pmconne — CHANGES_REQUESTED** [review]
Documentation comments will appear on our public APi reference. Please document clearly and thoroughly using complete sentences, avoiding abbreivations, etc. Link to related APIs - e.g., add to `ViewCreator2d` something like `@see [[ViewCreator3d]] to create a view for a 3d model.`.

**Comment by pmconne** [comment]
@roopksaini do you plan to finish up this PR?

**Comment by roopksaini** [comment]
@pmconne Updated. Kindly review.

**Review by pmconne — APPROVED** [review]

**Pull request merged**